program/tests/common.rs

```diff
@@ -0,0 +1,93 @@
+#![allow(dead_code)]
+#![cfg(feature = "test-sbf")]
+
+use {
+    address_lookup_table::state::{AddressLookupTable, LookupTableMeta},
+    solana_program_test::*,
+    solana_sdk::{
+        account::AccountSharedData,
+        instruction::{Instruction, InstructionError},
+        pubkey::Pubkey,
+        signature::{Keypair, Signer},
+        transaction::{Transaction, TransactionError},
+    },
+    std::borrow::Cow,
+};
+
+pub async fn setup_test_context() -> ProgramTestContext {
+    let mut program_test = ProgramTest::default();
+    program_test.prefer_bpf(false);
```

Comment:
Is there a reason for not using BPF? We need to make sure that this works for BPF to be sure it'll do what's intended

Reply:
I can use BPF, yeah, but we still need to use an arbitrary address until https://github.com/solana-labs/solana/pull/35233 is available in `2.0`.

Reply:
no problem!